### ISSUE TITLE
youtube_dl.utils.ExtractorError - can't download embed-only video

### ISSUE THREAD

**Issue by pilotinheels** [ytdl-org/youtube-dl#27653]
## Checklist

- [ ] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2021.01.03**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
[01/03/21 15:39:35] [debug] System config: []
[01/03/21 15:39:35] [debug] User config: []
[01/03/21 15:39:35] [debug] Custom config: []
[01/03/21 15:39:35] [debug] Command-line args: ['--newline', '-i', '-o', 'C:\\Users\\tiffa\\Dropbox\\My PC (DESKTOP-02M7SHU)\\Downloads\\%(title)s.%(ext)s', '-v', '--ignore-config', '--hls-prefer-native', '-v', '--add-header', 'Origin: https://player.vimeo.com/video/467431044?dnt=1&app_id=122963', 'https://player.vimeo.com/video/467431044?dnt=1&app_id=122963']
[01/03/21 15:39:35] [debug] Adding header from command line option Origin: https://player.vimeo.com/video/467431044?dnt=1&app_id=122963
[01/03/21 15:39:35] [debug] Encodings: locale cp1252, fs mbcs, out cp1252, pref cp1252
[01/03/21 15:39:35] [debug] youtube-dl version 2021.01.03
[01/03/21 15:39:35] [debug] Python version 3.4.4 (CPython) - Windows-10-10.0.19041
[01/03/21 15:39:35] [debug] exe versions: ffmpeg 3.3.2, ffprobe 3.3.2
[01/03/21 15:39:35] [debug] Proxy map: {}
[01/03/21 15:39:35] ERROR: Cannot download embed-only video without embedding URL. Please call youtube-dl with the URL of the page that embeds this video.
[01/03/21 15:39:35] Traceback (most recent call last):
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\extractor\common.py", line 632, in _request_webpage
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\YoutubeDL.py", line 2248, in urlopen
[01/03/21 15:39:35]   File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
[01/03/21 15:39:35]   File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
[01/03/21 15:39:35]   File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
[01/03/21 15:39:35]   File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
[01/03/21 15:39:35]   File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default
[01/03/21 15:39:35] urllib.error.HTTPError: HTTP Error 403: Forbidden
[01/03/21 15:39:35] 
[01/03/21 15:39:35] During handling of the above exception, another exception occurred:
[01/03/21 15:39:35] 
[01/03/21 15:39:35] Traceback (most recent call last):
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\extractor\vimeo.py", line 595, in _real_extract
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\extractor\common.py", line 665, in _download_webpage_handle
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\extractor\common.py", line 650, in _request_webpage
[01/03/21 15:39:35] youtube_dl.utils.ExtractorError: Unable to download webpage: HTTP Error 403: Forbidden (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
[01/03/21 15:39:35] Traceback (most recent call last):
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\extractor\common.py", line 632, in _request_webpage
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\YoutubeDL.py", line 2248, in urlopen
[01/03/21 15:39:35]   File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
[01/03/21 15:39:35]   File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
[01/03/21 15:39:35]   File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
[01/03/21 15:39:35]   File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
[01/03/21 15:39:35]   File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default
[01/03/21 15:39:35] urllib.error.HTTPError: HTTP Error 403: Forbidden
[01/03/21 15:39:35] 
[01/03/21 15:39:35] During handling of the above exception, another exception occurred:
[01/03/21 15:39:35] 
[01/03/21 15:39:35] Traceback (most recent call last):
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\extractor\vimeo.py", line 595, in _real_extract
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\extractor\common.py", line 665, in _download_webpage_handle
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\extractor\common.py", line 650, in _request_webpage
[01/03/21 15:39:35] youtube_dl.utils.ExtractorError: Unable to download webpage: HTTP Error 403: Forbidden (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
[01/03/21 15:39:35] 
[01/03/21 15:39:35] During handling of the above exception, another exception occurred:
[01/03/21 15:39:35] 
[01/03/21 15:39:35] Traceback (most recent call last):
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\YoutubeDL.py", line 803, in wrapper
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\YoutubeDL.py", line 824, in __extract_info
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\extractor\common.py", line 532, in extract
[01/03/21 15:39:35]   File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp7n_m4vnl\build\youtube_dl\extractor\vimeo.py", line 605, in _real_extract
[01/03/21 15:39:35] youtube_dl.utils.ExtractorError: Cannot download embed-only video without embedding URL. Please call youtube-dl with the URL of the page that embeds this video.
[01/03/21 15:39:35] 

```

## Description

Was downloading some videos with a login wall, things were great until it stopped working.
I updated Youtube-DL at this point, tried again and it wasn't working. I since cleared the log unfortunately because I did a LOT of things, but I believe it was a very similar error about an embedded video.
I did a lot of researching on reddit and on here for previous issues for embed only videos. I tried things like -refer and "origin: URL" from here: https://www.reddit.com/r/youtubedl/comments/cvvfy8/private_vimeo_video_problem_maybe_my_commands_are/ and other places.
I tried also playing the video and opening up the developer tools, right clicking on the video link and any content playing in the Network tab, Copying Header, pasting into a text file and looking for something called Origin as suggested in some other posts, and tried to find the m3u4 code/ID or whatever it is.
I also added my login info for the site, which didn't work.
I cleared my Youtube-dl cache, as well.
I'm not sure what else to do. It looks like when I open the actual Vimeo link it says it's a private video, which makes sense.
This account is something I can't give out, as it's to a website with course videos on it, but I can assure you that the videos work in the browsers.

Feel free to ask me anything, I am very n00b at this so if you can would you make your explanations very simple? Think ELI5 haha! Thank you so so much for any help!

**Comment by dstftw** [comment]
You must pass the original URL of the page that embeds the video.

**Comment by pilotinheels** [comment]
Whoops!

Do you want me to open a new case or just edit my current one?

The original url: 
https://member.ruthsoukup.com/lessons/4-design-your-essential-assets/
The vimeo URL:
https://player.vimeo.com/video/467431044?dnt=1&app_id=122963
